src/tests/nativeaot/SmokeTests/PInvoke/PInvokeNative.cpp

```diff
@@ -572,7 +572,8 @@ DLL_EXPORT bool __stdcall InlineArrayTest(inlineStruct* p, inlineUnicodeStruct *
     if (CompareAnsiString(p->inlineString, "Hello") != 1)
        return false;
 
-    if (!VerifyUnicodeString(q->inlineString))
+    unsigned short expected[] = { 'H', 'e', 'l', 'l', 'o', ' ', 'W', 'o', 'r', 'l', 0 };
```

Comment:
The test was validating wrong behavior. The test failed on CoreCLR when run before this change.